docs/checks/generalizedCoastlineCheck.md

```diff
@@ -0,0 +1,18 @@
+# Generalized Coastline Check
+
+This check flags any LineItem that has the `natural=coastline` tag, and also has a percentage of [Segment](https://github.com/osmlab/atlas/blob/dev/src/main/java/org/openstreetmap/atlas/geography/)s
```

Comment:
Do you mind adding for context to this document? Mainly, can you explain what the underlying problem is and why this check should exist?

Reply:
Done!